tools/travis/build.sh

```diff
@@ -11,7 +11,7 @@ tools/build/scanCode.py .
 
 cd $ROOTDIR/ansible
 
-ANSIBLE_CMD="ansible-playbook -i environments/travis"
+ANSIBLE_CMD="ansible-playbook -i environments/local"
```

Comment:
Ah good to know, it would be good in the future to dynamically get this value, or maybe find a way to just use localhost with any port required expose on localhost.

The PR is green in Travis, so no more comments, PR is ready